src/coll/mod.rs

```diff
@@ -47,8 +48,8 @@ const MAX_INSERT_DOCS_BYTES: usize = 16 * 1000 * 1000;
 /// # use bson::{bson, doc};
 /// # use mongodb::{Client, error::Result};
 /// #
-/// # fn start_workers() -> Result<()> {
-/// # let client = Client::with_uri_str("mongodb://example.com")?;
+/// # async fn start_workers() -> Result<()> {
```

Comment:
these examples show using the various objects across threads. Now that we're async, these should probably be using tasks, but I wasn't sure how to represent that in a runtime agnostic way.

Reply:
That's a good question. For now, let's leave it, but can we file a ticket so we can revisit before releasing?

Reply:
filed [RUST-329](https://jira.mongodb.org/browse/RUST-329)